app/controllers/my_facilities_controller.rb

```diff
@@ -96,7 +90,11 @@ def set_time_zone
     Time.use_zone(time_zone) { yield }
   end
 
-  def authorize_my_facilities
-    authorize(:dashboard, :view_my_facilities?)
+  def authorize_view
```

Comment:
This is this radical simplicity of the read permissions, the following two functions are all we need for all reporting / analytics related permissions everywhere.